File: src/exercise2/go/swindle.go
package main

import (
	"time"
)

func main() {
	backend := Backend{NewDatabase()}
	userId := 1

	backend.onAddClick(userId, time.July)
	backend.onAddClick(userId, time.July)
	backend.onAddClick(userId, time.July)
	backend.onAddClick(userId, time.July)
	backend.onAddClick(userId, time.July)
	backend.onAddClick(userId, time.August)
	backend.onAddClick(userId, time.August)
	backend.onAddClick(userId, time.August)
	backend.onAddClick(userId, time.August)

	backend.cashOut(userId, time.July)
	backend.cashOut(userId, time.August)
}

File: src/exercise2/java/Swindle.java
import java.time.Month;

public class Swindle {

    public static void main(String[] args) {
        Backend backend = new Backend();
        int userId = 1;

        backend.onAddClick(userId, Month.JULY);
        backend.onAddClick(userId, Month.JULY);
        backend.onAddClick(userId, Month.JULY);
        backend.onAddClick(userId, Month.JULY);
        backend.onAddClick(userId, Month.JULY);
        backend.onAddClick(userId, Month.AUGUST);
        backend.onAddClick(userId, Month.AUGUST);
        backend.onAddClick(userId, Month.AUGUST);
        backend.onAddClick(userId, Month.AUGUST);

        backend.cashOut(userId, Month.JULY);
        backend.cashOut(userId, Month.AUGUST);
    }
}
